packages/dd-trace/test/span_processor.spec.js

```diff
@@ -42,6 +42,7 @@ describe('SpanProcessor', () => {
       sample: sinon.stub()
     }
     config = {
+      tracing: true,
```

Comment:
Why is this needed if it's the default? Is there a bug in the new code? I wonder if it's related to the large changes in benchmark results.